internal/helpers/helpers.go

```diff
@@ -11,3 +20,17 @@ func Dedupe(s []string) []string {
 	}
 	return ret
 }
+
+func MakeTimestamp() int64 {
+	return time.Now().UnixNano() / int64(time.Millisecond)
+}
+
+func FatalServiceError(err error, ignoreServiceErrors bool) {
+	if ld.IsTransient(err) {
```

Comment:
Nit: since this is checking for transient errors, maybe an alternative name is LogTransientError? Otherwise some may get confused the difference between `log.Error.Fatal` and `FatalServiceError`.